jbpm-console-ng-dashboard/jbpm-console-ng-dashboard-client/src/main/java/org/jbpm/dashboard/renderer/client/panel/ProcessDashboard.java

```diff
@@ -1,13 +1,13 @@
 /**
  * Copyright (C) 2015 Red Hat, Inc. and/or its affiliates.
-
- *
+ * <p/>
+ * <p/>
```

Comment:
HTML tags should not be part of the license header. I think the got here because the header starts with "/**" instead of "/*". Could you please check this and fix the header?